Lib/test/test_posix.py

```diff
@@ -272,6 +272,30 @@ def test_pread(self):
         finally:
             os.close(fd)
 
+    @unittest.skipUnless(hasattr(posix, 'preadv'), "test needs posix.preadv()")
+    def test_preadv(self):
+        fd = os.open(support.TESTFN, os.O_RDWR | os.O_CREAT)
+        try:
+            os.write(fd, b'test1tt2t3')
+            os.lseek(fd, 0, os.SEEK_SET)
```

Comment:
Hum, is it really needed to seek back at offset 0? It would be more interesting to test preadv() with an offset different than the current offset.